vmware-event-router/hack/README.md

```diff
@@ -0,0 +1,76 @@
+# Deploy vCenter Event Broker Application to existing Kubernetes Cluster
+
+For customers with an existing Kubernetes Cluster, you can deploy the underlying components that make up the vCenter Event Broker Appliance. The instructions below will guide you in downloading the required files and using the `create_k8s_config.sh` shell script to aide in deploying the VEBA K8s application.
+
+The script will prompt users for the required input and  automatically setup and deploy both OpenFaaS and the VMware Event Router giving you a similar setup like the vCenter Event Broke Appliance.
```

Comment:
Can we make OpenFaaS optional? I suspect some users will find the script useful but already have OpenFaaS installed/want to customize OpenFaaS (persistency, scalability, etc.). Also, this might keep the possibility to add AWS EventBridge later/manually.

Reply:
Going to keep the logic simple for now and assume OpenFaaS will be part of the deployment. In theory, L100-106 can simply be commented out to prevent the deployment of OpenFaaS. I'd like to get some feedback and we can always adjust

Reply:
Just added a new option in script to not deploy OpenFaaS